### ISSUE TITLE
About Finetuning Stage 3 of English

### ISSUE THREAD

**Issue by GMago123** [Jacob-Zhou/gecdi#4]
您好，论文附录B.2里提到英文GEC模型调优的Stage3是在"W&I Locness test set”上进行的，这具体是什么意思呢？根据我的了解，WILocness的测试集是不开放的吧，是BEA19的评测数据集。请问是原文描述有误还是我理解错了呢？

**Comment by Jacob-Zhou** [comment]
非常感谢！确实是原文描述有误，正确的描述应该是 We further fine-tune the model on the W&I + LOCNESS train set only.
调优是在训练集上进行的。
